application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -2533,10 +2533,16 @@ public function add_quota($sSessionKey, $iSurveyID, $sQuotaName, $iLimit, $bActi
 
                 switch ($sAction) {
                     case 'terminate':
-                        $iAction = Quota::ACTION_TERMINATE;
+                        $iAction = Quota::TERMINATE_AFTER_ALL_VISIBLE_QUOTA_QUESTION;
+                        break;
+                    case 'terminate_all':
+                        $iAction = Quota::TERMINATE_AFTER_ALL_QUOTA_QUESTIONS;
+                        break;
+                    case 'terminate_always':
+                        $iAction = Quota::TERMINATE_AFTER_ALL_GROUPS;
```

Comment:
`Quota::TERMINATE_AFTER_ALL_STEPS;` or `Quota::TERMINATE_AFTER_ALL_ACTION;` ?

Since we have question by question mode